pkg/api/handlers/libpod/networks.go

```diff
@@ -44,6 +44,31 @@ func CreateNetwork(w http.ResponseWriter, r *http.Request) {
 	utils.WriteResponse(w, http.StatusOK, report)
 }
 
+func UpdateNetwork(w http.ResponseWriter, r *http.Request) {
+	if v, err := utils.SupportedVersion(r, ">=4.0.0"); err != nil {
+		utils.BadRequest(w, "version", v.String(), err)
+		return
+	}
+
+	runtime := r.Context().Value(api.RuntimeKey).(*libpod.Runtime)
+	ic := abi.ContainerEngine{Libpod: runtime}
+
+	networkUpdateOptions := entities.NetworkUpdateOptions{}
+	if err := json.NewDecoder(r.Body).Decode(&networkUpdateOptions); err != nil {
```

Comment:
Why are we doing JSON in the body? Just about everything else uses standard query parameters.

Reply:
I don't know what else might be added in future and update body in json format seemed easy to use and scale but I can switch to http parameter if that's a preference.